### DISCUSSION TITLE
Renovate not updating my packages (pnpm)

### DISCUSSION THREAD

**Discussion by mxz7** [renovatebot/renovate#19793] [Request Help] [answered]
I've had renovate running on my repo for a couple of weeks now and it was working at first, but it isn't updating any packages that I know 100% have had updates.

`pnpm update` output:
![Screenshot 2023-01-11 at 16 19 13](https://user-images.githubusercontent.com/29782397/211863543-88433f36-2356-4af2-af8a-6f6848bf3d90.jpg)

I can't find anything of notice in the job logs, any help with this?

my config is [here](https://github.com/tekoh/nypsi/blob/main/renovate.json)

**Comment by rarkins** [comment, marked as answer]
It might be because you're using ranges and the new versions satisfy the ranges? i.e. do these updates require changes to the `package.json` or only the lock file?
